assembl/views/api2/discussion.py

```diff
@@ -511,37 +511,56 @@ def extract_taxonomy_csv(request):
     import assembl.models as m
     discussion = request.context._instance
     db = discussion.db
-    format = get_format(request)
     extracts = db.query(m.Extract).all()
     extract_list = []
-    fieldnames = ["Thematic", "Message", "Content Harvested", "Qualify by nature", "Qualify by action",
+    fieldnames = ["Thematic", "Message", "Content harvested", "Qualify by nature", "Qualify by action",
                   "Owner of the message", "Published on", "Harvester", "Harvested on", "Nugget"]
     for extract in extracts:
         thematic = db.query(m.Idea).filter(m.Idea.id == extract.idea_id).first()
+        if thematic:
+            if thematic.title:
+                thematic = thematic.title.entries[0].value
```

Comment:
You will potentially have a mix of english and french if you do that.
You should use `thematic.title.best_lang(user_prefs).value`, see global_vote_results_csv in assembl/views/api2/votes.py for an example.